blosc/frame.c

```diff
@@ -1043,7 +1043,7 @@ uint8_t* get_coffsets(blosc2_frame_s *frame, int32_t header_len, int64_t cbytes,
     // For in-memory frames, the coffset is just one pointer away
     uint8_t* off_start = frame->cframe + off_pos;
     if (off_cbytes != NULL) {
-      *off_cbytes = *(int32_t*) (off_start + BLOSC2_CHUNK_CBYTES);
+      *off_cbytes = sw32_(off_start + BLOSC2_CHUNK_CBYTES);
```

Comment:
Also prefer to re-read the entire blosc header here too.

Reply:
+1